docs/03-tutorials/00-api-exposure/apix-02-expose-workload-apigateway.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Expose a workload
+---
+
+This tutorial shows how to expose service endpoints and configure different allowed HTTP methods for them using API Gateway Controller.
+
+The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
```

Comment:
```suggestion
It may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
```

Reply:
Or: `You can use it as a follow up...` - to avoid the repetition of `this tutorial`.